src/backend/main.ts

```diff
@@ -398,36 +396,6 @@ if (!gotTheLock) {
 
     initTrayIcon(mainWindow)
 
-    // hotkey to reload the app
-    globalShortcut.register('CommandOrControl+R', () => {
-      mainWindow.reload()
-    })
-
-    // hotkey to quit the app
-    globalShortcut.register('CommandOrControl+Q', () => {
-      handleExit()
-    })
-
-    // hotkey to open the dev tools
-    globalShortcut.register('CommandOrControl+Shift+I', () => {
-      mainWindow.webContents.openDevTools()
-    })
-
-    // hotkey to open the settings on frontend
-    globalShortcut.register('CommandOrControl+K', () => {
-      sendFrontendMessage('openScreen', '/settings/app/default/general')
-    })
-
-    // hotkey to open the library screen on frontend
-    globalShortcut.register('CommandOrControl+L', () => {
-      sendFrontendMessage('openScreen', '/library')
-    })
-
-    // hotkey to open the downloads screen on frontend
-    globalShortcut.register('CommandOrControl+J', () => {
-      sendFrontendMessage('openScreen', '/download-manager')
```

Comment:
I wonder if we can add these shortcuts in the frontend shortcuts.ts file https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/blob/main/src/frontend/helpers/shortcuts.ts

that uses event listeners of the frontend so we can listen the combinations without having to deal with the menu or depending on electron features

Reply:
we can always call electron functions when needed through window.api to still support the quit/devtools/reload commands

Reply:
I made another PR with that idea since it's reusing things removed from this PR https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/2684

we can keep all the new shortcuts while also making them local without relying on electron accelerators